concordia/views.py

```diff
@@ -1402,13 +1402,17 @@ def reserve_asset(request, *, asset_pk):
             if reservation.tombstoned:
                 if reservation.reservation_token == reservation_token:
                     am_i_tombstoned = True
+                    logger.debug("I'm tombstoned")
```

Comment:
Should we work the asset / user / reservation ID into these in case we need to use them on a running system?

```suggestion
                    logger.debug("I'm tombstoned %s", reservation)
```